dd-sdk-android-core/src/main/kotlin/com/datadog/android/api/net/RequestFactory.kt

```diff
@@ -17,12 +17,14 @@ fun interface RequestFactory {
     /**
      * Creates a request for the given batch.
      * @param context Datadog SDK context.
+     * @param executionContext Gives information about the execution context of this and previous requests.
```

Comment:
```suggestion
     * @param executionContext Information about the execution context of this and previous requests.
```
Also to avoid confusion we must say that it is for the current batch only (otherwise it is not clear if it is for the previous batch as well).